src/GameKit/GKLeaderboardSet.cs

```diff
@@ -0,0 +1,60 @@
+//
+// GKLeaderBoardSet.cs: Non-generated API for GKLeaderBoard.
+//
+// Authors:
+//   Whitney Schmidt (whschm@microsoft.com)
+//
+// Copyright 2020, Microsoft Corp. All rights reserved.
+//
+// The class can be either constructed from a string (from user code)
+// or from a handle (from iphone-sharp.dll internal calls).  This
+// delays the creation of the actual managed string until actually
+// required
```

Comment:
Is this part copied from somewhere else? It doesn't look like it's relevant to this file.

Reply:
Removed.